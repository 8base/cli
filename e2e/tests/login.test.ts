import { runner } from '../runner';

it('As a user, I can use help flag for see help information for `login` command.', async () => {
  const { stdout } = await runner()('login', '--help');

  expect(stdout).toMatchSnapshot();
});

// it('As a user, I can use `login` command for login.', async () => {
//   await runner()('config', '-s=https://prestaging-api.8basedev.com/');

//   const { stdout } = await runner()('login', '-e=zouxuoz-8base+7414205639@yandex.ru', '-p=RsqKchqXqE>dKk9ZQda6');

//   expect(stdout.replace(/Time estimate: \d+ ms\./, 'Time estimate: 100 ms.')).toMatchSnapshot();
// });