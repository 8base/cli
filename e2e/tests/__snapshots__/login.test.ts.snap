// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`As a user, I can use help flag for see help information for \`login\` command. 1`] = `
"8base login [OPTIONS]

Options:
  -e, --email     user email                                            [string]
  -p, --password  user password                                         [string]"
`;
